src/ORM/Connect/DBConnector.php

```diff
@@ -52,7 +52,7 @@ protected function databaseError($msg, $errorLevel = E_USER_ERROR, $sql = null,
 
         // Format query if given
         if (!empty($sql)) {
-            $formatter = new SQLFormatter();
+            $formatter = SQLFormatter::create();
```

Comment:
Unrelated - but since `SQLFormatter` explicitly uses the `Injector` trait I'm using `::create()` below - so this should also use it.